cli/internal/cmd/iamcreate.go

```diff
@@ -0,0 +1,21 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// NewIAMCreateCmd returns a new cobra.Command for the iam create parent command. It needs another verb, and does nothing on its own.
+func newIAMCreateCmd() *cobra.Command {
```

Comment:
Agreed! I discussed with Leo but we wanted to keep the 1 command / file structure. Maybe we should discuss the structure of this folder generally. It might make sense to somehow group parents with their subcommands?